site/src/components/LicenseBanner/LicenseBanner.tsx

```diff
@@ -1,19 +1,9 @@
-import { useActor } from "@xstate/react"
-import { useContext, useEffect } from "react"
-import { XServiceContext } from "xServices/StateContext"
+import { useDashboard } from "components/Dashboard/DashboardProvider"
 import { LicenseBannerView } from "./LicenseBannerView"
 
 export const LicenseBanner: React.FC = () => {
-  const xServices = useContext(XServiceContext)
-  const [entitlementsState, entitlementsSend] = useActor(
-    xServices.entitlementsXService,
-  )
-  const { errors, warnings } = entitlementsState.context.entitlements
-
-  /** Gets license data on app mount because LicenseBanner is mounted in App */
-  useEffect(() => {
-    entitlementsSend("GET_ENTITLEMENTS")
-  }, [entitlementsSend])
+  const { entitlements } = useDashboard()
+  const { errors, warnings } = entitlements
```

Comment:
NIT: you can
````
  const { entitlements: { errors, warnings }} = useDashboard()
````
but totally up to you if you find that more readable or not. Sometimes the linter annoyingly wraps the lines so it's not always a space-saver

Reply:
Sure thing!